src/utils/helpers/use-media-query/useMediaQuery.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export default function useMediaQuery(queryInput) {
+  const query = queryInput.replace(/^@media( ?)/m, '');
+  const supportMatchMedia = typeof window !== 'undefined' && typeof window.matchMedia !== 'undefined';
```

Comment:
do we need to check that? Afaik it has full support in all browsers.

Reply:
Yeah you're right, looks like it is https://caniuse.com/matchmedia - I'll take that check out